Expecto/Expecto.fs

```diff
@@ -454,7 +454,18 @@ module Impl =
       | Passed -> "Passed"
       | Ignored reason -> "Ignored: " + reason
       | Failed error -> "Failed: " + error
-      | Error e -> "Exception: " + e.ToString()
+      | Error e -> 
+        let rec printInner (ex: exn) msg =
+          let currentMsg = 
+            msg + (sprintf "%s%s" Environment.NewLine (ex.ToString()))
+          if isNull ex.InnerException then
+            currentMsg
+          else
+            printInner ex.InnerException currentMsg
+
+        let exMsg = printInner e ""
+        
+        "Exception: " + exMsg
```

Comment:
Join together to `"Exception: " + printInner e ""`

Reply:
Done